### PR TITLE
Add link for mobx-preact

### PR DESCRIPTION
**Pull request by philmander**
...which I've just updated and is based on `mobx-react`

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -367,9 +367,7 @@
 * [MobX react 开发者工具](https://github.com/mobxjs/mobx-react-devtools)
 * [MobX remotedev: 使用 MobX 版本的 Redux 开发工具](https://github.com/zalmoxisus/mobx-remotedev)
 * [MobX inferno 绑定](https://www.npmjs.com/package/mobx-inferno)
-* 对于 Preact，可以使用
-  [preact-compat](https://github.com/developit/preact-compat) 模块 + mobx-react
-  。
+* [MobX preact 绑定](https://www.npmjs.com/package/mobx-preact)
```
Comment (hawkins) [inline README-CN.md]: Hey, could you make this a link to the repository on github or such please? :)

File: README.md [hunk 2]
```diff
@@ -377,8 +377,7 @@ Your contributions and suggestions are heartily welcome. =^.^=
 * [MobX react bindings](https://github.com/mobxjs/mobx-react)
 * [MobX remotedev: Use the Redux Devtools with MobX](https://github.com/zalmoxisus/mobx-remotedev)
 * [MobX inferno bindings](https://www.npmjs.com/package/inferno-mobx)
-* For Preact the [preact-compat](https://github.com/developit/preact-compat)
-  module + mobx-react can be used.
+* [MobX preact bindings](https://www.npmjs.com/package/mobx-preact)
```
Comment (hawkins) [inline README.md]: See other comment: 

> Hey, could you make this a link to the repository on github or such please? :)

**Comment by philmander** [comment]
Oh sorry, was just following the Inferno example above it. Updated now.

**Comment by hawkins** [comment]
No problem, it's a recent rule we haven't yet retroactively applied to older links. 😄 

Thanks for updating the awesome list and the package! 🙌

**Pull request merged**